api/URL.json

```
@@ -145,10 +145,14 @@
               "version_added": "14"
             },
             "safari": {
-              "version_added": "6"
+              "version_added": "6",
+              "partial_implementation": true,
+              "notes": "In Safari 14 and earlier, calling the URL constructor with a base URL whose value is undefined causes Safari to throw a type error; see <a href='https://webkit.org/b/216841'>WebKit bug 216841</a>."
```

Suggestion: if this applies to all versions, let's not specify the version (so it doesn't go stale if this doesn't get fixed in the next release)

Style: more code styling! Why not?

```suggestion
              "notes": "Calling the <code>URL</code> constructor with a base URL whose value is <code>undefined</code> causes Safari to throw a <code>TypeError</code>; see <a href='https://webkit.org/b/216841'>WebKit bug 216841</a>."
```

---

Ugh, I'm sorry @sideshowbarker. I just got to the other tab I opened with #6749. I see now that it's probably fixed in whatever the next release of Safari is. That's kind of annoying—I would love to say something a bit simpler, like "Before Safari 14.1" or something. But since we don't know what the next version of Safar is going to be…

```suggestion
              "notes": "In Safari 14 and earlier, calling the <code>URL</code> constructor with a base URL whose value is <code>undefined</code> causes Safari to throw a <code>TypeError</code>; see <a href='https://webkit.org/b/216841'>WebKit bug 216841</a>."
```